plugins/hbase_logs.yaml

```diff
@@ -0,0 +1,132 @@
+version: 0.0.1
+title: Apache HBase
+description: Log parser for Apache HBase
+parameters:
+  - name: enable_master_log
+    type: bool
+    default: true
+  - name: master_log_path
+    type: "[]string"
+    default: 
+    - "/usr/local/hbase*/logs/hbase*-master-*.log"
+  - name: enable_region_log
+    type: bool
+    default: true
+  - name: region_log_path
+    type: "[]string"
+    default: 
+    - "/usr/local/hbase*/logs/hbase*-regionserver-*.log"
+  - name: enable_zookeeper_log
+    type: bool
+    default: false
+  - name: zookeeper_log_path
+    type: "[]string"
+    default: 
+    - "/usr/local/hbase*/logs/hbase*-zookeeper-*.log"
+  - name: start_at
+    type: string
+    supported:
+      - beginning
+      - end
+    default: end
+template: |
+  receivers:
+    {{ if .enable_master_log }}
+    filelog/master:
```

Comment:
This receiver should have a 
```yaml
attributes:
  log_type: hbase.master
```
section in it.

Reply:
I saw this on some other plugins but did not add it to mine, oops. Just went in and made the change for all three sections.